blueocean-dashboard/src/main/js/jenkins-js-extension.yaml

```diff
@@ -2,8 +2,8 @@
 # NB: "component" currently maps to modules, not "symbols" so make sure to "export default"
 # WARNING: If you change this you'll have to change io.jenkins.blueocean.jsextensions.JenkinsJSExtensionsTest as well :(
 extensions:
-  - component: AdminNavLink
-    extensionPoint: jenkins.logo.top
```

Comment:
It does but this was its only usage in the "blueocean-plugin" repo. Perhaps we should remove it?